src/main/java/uk/gov/hmcts/darts/utilities/deserializer/LocalDateTimeTypeDeserializer.java

```diff
@@ -0,0 +1,22 @@
+package uk.gov.hmcts.darts.utilities.deserializer;
+
+import com.fasterxml.jackson.core.JacksonException;
+import com.fasterxml.jackson.core.JsonParser;
+import com.fasterxml.jackson.databind.DeserializationContext;
+import com.fasterxml.jackson.databind.JsonDeserializer;
+
+import java.io.IOException;
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+
+public class LocalDateTimeTypeDeserializer extends JsonDeserializer<LocalDateTime> {
+
+    private static final DateTimeFormatter FORMATTER = DateTimeFormatter.ofPattern("d:MMM:uuuu HH:mm:ss");
```

Comment:
Is this really the date format? Looks like it's got colon delimiters like in the time part?